_plugins/auto_logs_markup.rb

```diff
@@ -26,9 +26,19 @@
 
 NON_BREAKING_SPACE = '&nbsp;'
 
+COLORS = %w(aqua aquamarine blue blueviolet brown cadetblue chartreuse chocolate
+coral cornflowerblue crimson cyan darkblue darkcyan firebrick forestgreen
+fuchsia gold goldenrod green grey hotpink indianred indigo khaki lawngreen
+magenta maroon mediumblue mediumpurple mediumseagreen navy olive orange orchid
+papayawhip peachpuff peru pink plum purple rebeccapurple red rosybrown royalblue
+salmon seagreen seashell sienna silver skyblue slateblue springgreen steelblue
+tan teal thistle tomato turquoise violet wheat yellow yellowgreen).freeze
+
 # Convert logs from plain text to HTML with line number links.
 #
 Jekyll::Hooks.register :documents, :pre_render do |post|
+  # Reset color data for each post.
+  colors, color_index, name_colors = COLORS.shuffle, -1, {}
```

Comment:
Why shuffle? I'd prefer to keep this deterministic.

Reply:
The idea was so people aren't stuck with the same color in every post.

Reply:
The idea was so people aren't stuck with the same color in every post, and also, the colors would be based on order of posting.

Reply:
The idea was so people aren't stuck with the same color in every post, and the colors would be based on order of posting in the first post parsed, and also the cumulative number of participants is higher, so if we don't reset, some colors might be used by more than one participant in a meeting.